internal/loadgen/eventhandler/transport.go

```diff
@@ -49,12 +53,24 @@ func (t *Transport) SendV2Events(ctx context.Context, r io.Reader, ignoreErrs bo
 }
 
 func (t *Transport) sendEvents(req *http.Request, r io.Reader, ignoreErrs bool) error {
+	t.logger.Debug("sending request")
 	res, err := t.client.Do(req)
 	if err != nil {
-		return err
+		return fmt.Errorf("cannot complete request: %w", err)
```

Comment:
Are we then logging this when it's bubbled up?

Reply:
yes, errors bubble up until they are logged in `apmsoak/cmd/run.go`, https://github.com/elastic/apm-perf/blob/3a6d6182717a2bfeff355eebeec1c62b7f7d12a8/cmd/apmsoak/run.go#L73